### DISCUSSION TITLE
NodePort Services reachable with just control plane VIP?

### DISCUSSION THREAD

**Discussion by markmaker** [kube-vip/kube-vip#967] [Q&A] [answered]
Hi,

sorry I'm still quite new to this stuff. :sweat_smile: 

The documentation mentions high availability for the control plane, and for the `LoadBalancer` services, if `svc_enable` is true. So far so clear.

What happens if we have `Service`s with `NodePort` exposure? Will they be reachable via just the control plane VIP?

I'm aware that this might involve extra hops (via `kube-proxy` if I understand correctly), but it might still be a viable solution if floating IPs are scarce and/or expensive, e.g. for public IP4 addresses on bare metal. Am I missing something?

_Mark

**Comment by thebsdbox** [comment, marked as answer]
A service of type `NodePort` will be ignored by kube-vip, it's watcher is only for `LoadBalancer` service types.
